src/main/java/io/supertokens/mfa/Mfa.java

```diff
@@ -0,0 +1,57 @@
+package io.supertokens.mfa;
+
```

Comment:
need to also delete all the mfa data for a user when deleting the user. This change will be done in the authrecipe.java file.

This will also require a new plugin interface function wherein you can delete all the info for a user in the mfa table (this will be a appIdentifier input and not a tenantIdentifier input)

Reply:
Done.